#!/usr/bin/env python
PACKAGE = "roverbot"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("sonar_freq",    int_t,    0, "integer parameter for sonar freq", -10,  -500, 25)
gen.add("imu_freq", 	int_t, 	   0, "integer parameter for imu freq",    -10, -500,   1000)
gen.add("battery_freq", 	int_t, 	   0, "integer parameter for battery freq",    -10, -500,   25)
gen.add("cam_freq",    int_t,    0, "integer parameter for imu freq", -10, -500,   30)
gen.add("cam_on_off",   bool_t,   0, "A Boolean parameter for cam switch",  False)


exit(gen.generate(PACKAGE, "roverbot", "Roverbot"))
